app/templates/team.hbs

```diff
@@ -51,7 +51,7 @@
   <p class="text-center">
     The Ember CLI core team is responsible for maintaining ember-cli, the command line interface for managing and packaging
     Ember.js applications and addons. The team also maintains many of the addons in the default blueprint as well as
-    fastboot.
+    Fastboot and Ember Engines.
```

Comment:
while it might be true that some people on the CLI team maintain those addons together with other people, I'm not sure it's correct to say that "the CLI team" maintains them. I personally am not maintaining them or even using them, and don't want the additional pressure of having to maintain those addons too. Since they are not directly related to Ember CLI I'm not sure that the CLI team should even be responsible for them. If they are "official" addons then maybe we should have dedicated teams for them 🤔

Reply:
@Turbo87 I agree this is a misrepresentation where engines are concerned, and perhaps fastboot as well. Personally, I'm in the awkward position of helping to maintain engines but not currently using them, so I can relate to your perspective.

It's worth discussing whether these projects should have their own "official" teams and representation on this page. Certainly the CLI team should not be assigned more responsibility than they want.

Reply:
After this, I super agree with you guys! @dgeb @Turbo87

Fastboot & Engines are official projects of Ember core team, but the responsibility is not clear of wich team members are dedicated on this. Can be very cool we having  a "Ember Engines Core Team" & "Fastboot Core Team"

Reply:
Ya, I don't think this particular edit is needed. Can we drop the changes to this file and land  the other change?

Reply:
yup @rwjblue, I'll do it

Reply:
done @rwjblue @jenweber